### PR TITLE
Reduce "global_common" coupling by moving methods that could be static onto "semantically-closer" py files

### PR DESCRIPTION
**Pull request by deeplow**
Based on work initially made by @gmarmstrong on https://github.com/freedomofpress/dangerzone/pull/166:
  - moves container-specific code out of `global_common.py` and into `container.py`
  - creates a `util.py` for static methods used through the whole app
  - move banner code from `global_common` onto `cli.py` given that it's only displayed there
  - updates tests to reflect these changes
  - move `ocr_languages` from `global_common` onto its own `json` file in `share/ocr-languages.json` to simplify `global_common` logic

### CODE REVIEW

**Comment by deeplow** [comment]
@gmarmstrong let me know what you think. I have limited the scope of this PR to moving things out of global common. 
The refactor to make static the code in `gui_common` will be in a future PR.

File: dangerzone/container.py [hunk 1]
```diff
@@ -29,6 +27,106 @@
...
+
+def get_runtime() -> str:
+    container_tech = get_runtime_name()
+    runtime = shutil.which(container_tech)
+    if runtime is None:
+        raise NoContainerTechException(container_tech)
+    return runtime
+
+
+def install() -> Optional[bool]:
```
Comment (gmarmstrong) [inline dangerzone/container.py:52]: I would either return `bool`, or keep the `Optional` but raise an exception (to be caught) upon failure to install.

File: dangerzone/gui/main_window.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def __init__(self, global_common: GlobalCommon) -> None:
         self.global_common = global_common
 
     def run(self) -> None:
-        self.global_common.install_container()
+        container.install()
```
Comment (gmarmstrong) [inline dangerzone/gui/main_window.py:107]: Check return value to ensure that the installation is successful.

File: dangerzone/cli.py [hunk 3]
```diff
@@ -90,7 +92,7 @@ def cli_main(
             exit(1)
 
     # Ensure container is installed
-    global_common.install_container()
+    container.install()
```
Comment (gmarmstrong) [inline dangerzone/cli.py:95]: Check return value to ensure that the installation is successful.

Comment (gmarmstrong) [reply dangerzone/cli.py:95]: I see that I didn't do that either, so maybe I'm just forgetting something here. Up to you!

Comment (gmarmstrong) [reply dangerzone/cli.py:95]: That may also just be outside the scope of this PR.

Comment (gmarmstrong) [reply dangerzone/cli.py:95]: Oh, never mind all of this! You added that "container-tech-checking logic" elsewhere. In that case, I'd just change the comment now to say "install container".

**Review by gmarmstrong — APPROVED** [review]
Looks great!

File: dangerzone/container.py (hunk 1 above)
Comment (deeplow) [reply dangerzone/container.py]: Changed to bool. I have an issue already for handling container installation failure https://github.com/freedomofpress/dangerzone/issues/193.

**Comment by deeplow** [comment]
Thanks for the review @gmarmstrong!

**Pull request merged**